backend/app/models/funder.rb

```diff
@@ -41,4 +42,8 @@ class Funder < ApplicationRecord
 
   scope :for_subgeographics, ->(abbreviations) { joins(:subgeographic_ancestors).where(subgeographics: {abbreviation: abbreviations}) }
   scope :for_geographics, ->(geographics) { joins(:subgeographic_ancestors).where(subgeographics: {geographic: geographics}) }
+  scope :with_projects_count, -> {
+    projects_count = Project.joins(:investments).where("investments.funder_id = funders.id").select("COUNT(DISTINCT investments.project_id)").to_sql
```

Comment:
I guess this could potentially be done without the join, just counting distinct project_ids on investments table

Reply:
🤔 I will check it @agnessa , thanks! I though that join to investment is required, because where contains `investments.funder_id` so I can obtain `projects_count` PER funder, but will check it one more time if it possible to get rid of it.